docs/conf.py

```diff
@@ -26,7 +26,6 @@
 
 extensions = ['sphinx.ext.autodoc',
               'sphinx.ext.doctest',
-              'sphinx.ext.pngmath',
```

Comment:
Is this no longer required for some reason?

Reply:
This prevents the website to build...
This is a sphinx issue... and as it seems to be required...

Reply:
https://github.com/sphinx-doc/sphinx/issues/3295

Reply:
try `imgmath`?

Reply:
it seems to be working !

Reply:
> This prevents the website to build...
> This is an issue on some Sphinx versions.. and as it seems to be required...

*as it seems NOT to be required....